### ISSUE TITLE
MocoOrientationTrackingGoal references can't be set from file 

### ISSUE THREAD

**Issue by nickbianco** [opensim-org/opensim-moco#624]
Since there is currently no `STOFileAdapter_<Rotation>`, there is no way to create a file to pass to MocoOrientationTrackingGoal.

**Comment by chrisdembia** [comment]
We should use TableProcessors.

**Comment by chrisdembia** [comment]
#572

**Comment by nickbianco** [comment]
I agree. It would be nice to have a TableOperator to handle packing and flattening. I think we've already discussed this option...
